common/changes/office-ui-fabric-react/fix--6984_2018-11-05-23-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Link: inherit font size and font weight.",
+      "type": "patch"
```

Comment:
OUFR API surface changed per the API file. @phkuo is this a breaking change to people who have created custom themes out there? (the new inherits property is required)

Reply:
Yes, but I think we point people to use the IPartialTheme instead anyways. All the required stuff in ITheme is just to verify at build time that our theming system has actually defined everything that's in a theme, so that downstream consumers can rely on all of it. So it should be fine. (The `typography` that's required in there is marked as experimental, for instance.)